pkg/config/constants/constants.go

```diff
@@ -150,4 +150,16 @@ const (
 	// CertProviderNone does not create any certificates for the control plane. It is assumed that some external
 	// load balancer, such as an Istio Gateway, is terminating the TLS.
 	CertProviderNone = "none"
+
+	// DataplaneMode namespace label for determining ambient mesh behavior
+	DataplaneMode        = "istio.io/dataplane-mode"
+	DataplaneModeAmbient = "ambient"
+
+	// AmbientRedirection specifies whether a pod has ambient redirection (to ztunnel) configured.
+	AmbientRedirection = "ambient.istio.io/redirection"
```

Comment:
I wonder if we need 'ambient.istio.io' vs just 'istio.io'. 

Also if 'istio.io/capture=ambientv1' isn't a better model ( with a backfill to add 'sidecarv1' ).

Reply:
That's also consistent with DataplaneMode.